### ISSUE TITLE
New oauth accounts get linked to the existing session's user

### ISSUE THREAD

**Issue by paolotiu** [nextauthjs/adapters#328]
### Title

New oauth accounts get linked to the existing session's user

### How to reproduce ☕️

https://github.com/paolotiu/next-auth-prisma-linking-repro

### Your question/bug report 📓

When a user is already signed in, but want to sign into a different oauth account, the current session's user gets linked to the newly created account.

https://user-images.githubusercontent.com/61054234/143861832-734e5cf1-2fae-406a-bab0-3bc10af74238.mp4

### Contributing  🙌🏽

Yes, I am

**Comment by balazsorban44** [comment]
I don't think this is Prisma specific. I'll move it into the core repo. This might need further investigation.
